libs/ui-components/src/components/AuthProvider/CreateAuthProvider/utils.ts

```
@@ -0,0 +1,533 @@
+
+  switch (a.type) {
+    case OrgAssignmentType.Static:
+      if (!isOrgAssignmentStatic(a) || !isOrgAssignmentStatic(b)) {
+        return false;
+      }
+      return (a.organizationName || '') === (b.organizationName || '');
+    case OrgAssignmentType.Dynamic:
+      if (!isOrgAssignmentDynamic(a) || !isOrgAssignmentDynamic(b)) {
+        return false;
+      }
+      const aClaimPath = a.claimPath || [];
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)